contacts/html-helper.js

```diff
@@ -0,0 +1,212 @@
+  newLi.id = person.listItemId;
+  parent.appendChild(newLi);
+  return newLi;
+};
+
+const createPlusButtonPersonHTML = (parent, person) => {
+  const plusButton = document.createElement(TAGS.BUTTON);
+  plusButton.id = person.plusButtonId;
+  plusButton.innerHTML = SYMBOLS.PLUS;
+  plusButton.custom = CUSTOMS.PLUS_BUTTON;
+  plusButton.disabled = isCountEqualToMax(person);
+  parent.appendChild(plusButton);
+};
+
+const createCountLikesPersonHTML = (parent, person) => {
+  const favCount = document.createElement(TAGS.DIV);
+  favCount.id = person.likesCountId;
+  favCount.className = CLASSES.FAVS_COUNT;
+  favCount.innerHTML = person.likes;
+  parent.appendChild(favCount);
+};
+
+const createMinusButtonPersonHTML = (parent, person) => {
+  const minusButton = document.createElement(TAGS.BUTTON);
+  minusButton.id = person.minusButtonId;
+  minusButton.innerHTML = SYMBOLS.MINUS;
+  minusButton.custom = CUSTOMS.MINUS_BUTTON;
+  minusButton.disabled = isCountEqualToMin(person);
+  parent.appendChild(minusButton);
+};
+
+const createAvatarImagePersonHTML = (parent, person) => {
+  const image = document.createElement(TAGS.IMAGE);
+  image.src = person.image;
+  image.alt = ALT.PIC;
+  image.className = CLASSES.AVATAR;
+  parent.appendChild(image);
+};
+
+const createCheckboxPersonHTML = (parent, person) => {
+  const checkbox = document.createElement(TAGS.INPUT);
+  checkbox.type = TYPES.CHECKBOX;
+  checkbox.custom = CUSTOMS.CHECKBOX;
+  checkbox.id = person.checkBoxId;
+  checkbox.disabled = false;
+  checkbox.checked = person.isChecked;
+  parent.appendChild(checkbox);
+};
+
+const createFullNamePersonHTML = (parent, person) => {
+  const fullName = document.createElement(TAGS.SPAN);
+  fullName.id = person.fullNameId;
+  fullName.className = CLASSES.FULL_NAME;
+  const firstName = document.createElement(TAGS.SPAN);
+  firstName.innerHTML = person.firstName;
+  firstName.className = CLASSES.MARGIN_RIGHT;
+  fullName.appendChild(firstName);
+  const lastName = document.createElement(TAGS.SPAN);
+  lastName.innerHTML = person.lastName;
+  fullName.appendChild(lastName);
+  parent.appendChild(fullName);
+};
+
+const createTrashPersonHTML = (parent, person) => {
+  const trash = document.createElement(TAGS.IMAGE);
+  trash.id = person.trashId;
+  trash.src = IMAGES.TRASH;
+  trash.alt = ALT.TRASH;
+  trash.custom = CUSTOMS.TRASH;
+  trash.className = CLASSES.TRASH;
+  parent.appendChild(trash);
+};
+
+const addToHTMLNewPerson = (person) => {
+  const newLi = createListItemPersonHTML(getElementById(DOCUMENT_ELEMENTS.CONTACTS.CONTACTS_LIST), person);
+  createPlusButtonPersonHTML(newLi, person);
+  createCountLikesPersonHTML(newLi, person);
+  createMinusButtonPersonHTML(newLi, person);
+  createAvatarImagePersonHTML(newLi, person);
+  createCheckboxPersonHTML(newLi, person);
+  createFullNamePersonHTML(newLi, person);
+  createTrashPersonHTML(newLi, person);
+  getElementById(DOCUMENT_ELEMENTS.CONTACTS.CONTACTS_LIST).appendChild(newLi);
+};
+
```

Comment:
You should have only one  blank line between functions. Please remove the extra ones.